src/ut/communicationmonitor_test.cpp

```diff
@@ -61,9 +62,11 @@ class CommunicationMonitorTest : public ::testing::Test
   virtual ~CommunicationMonitorTest()
   {
     cwtest_reset_time();
+    delete _am; _am = NULL;
   }
 
 private:
+  AlarmManager* _am;
```

Comment:
I think member variables of `_ma` and `_am` have pushed this over the limit. Can you find-replace it to _alarm_mgr and _mock_alarm?